ddp-workspace/projects/ddp-dsm-ui/src/app/FON/components/register-patients-modal/register-patients-modal.component.html

```diff
@@ -0,0 +1,90 @@
+<div class="dialogContainer">
+  <header>
+    <p>Register Patients</p>
+  </header>
+  <main>
+    <section>
+      <form [formGroup]="addPatientForm">
+        <ng-container formArrayName="patients">
+          <ng-container *ngFor="let patient of patients.controls; let i = index">
+            <div [formGroup]="patient" [ngClass]="{'hr': i > 0}" class="addPatientForm">
```

Comment:
Just wondering what 'hr' means ?

Reply:
Horizontal Rule. It displays a grey border on the top.